### ISSUE TITLE
Am I able to use a variable to "To:" field?

### ISSUE THREAD

**Issue by peluse** [dawidd6/action-send-mail#211]
Much in the same that these these do? If not may I request this as a feature?

          username: ${{secrets.MAIL_USERNAME}}
          password: ${{secrets.MAIL_PASSKEY}}

**Comment by dawidd6** [comment]
Sure, this is a GitHub Actions feature.

**Comment by peluse** [comment]
Thanks for the quick response!  Apparently the user of secrets works just fine but I found, with much trial an error, that any other kind of variable can't be passed as an input to the action.  I forked your repo (thanks, great stuff!) and tried a zillion thing.  What I think will work is using actions/github and pulling the email address in via a github API using the funciton below:

async function getActorEmail() {
  const octokit = new github.getOctokit()
  const { data } = await octokit.users.getByUsername({ username: core.getContext().actor })
  return data.email
}

However I'm totally new to thie Node.js stuff and simply adding "const github = require('@actions/github')" doesn't seem to work as I get an error about not finding @actions/github.  I've tried installatin in my workflow with:

    - name: Install Dependencies
        run: |
          npm install @actions/github
          
  juse before using the Action however it didn't help.  Reading about adding it in the Node.js proejct now.  Fun stuff, any advice would be super appreciated!
